File: API/Alliance-Bulletin-USS/Alliance-Bulletin-USS/Models/Bulletin.cs
using System;

namespace Alliance_Bulletin_USS.Models
{
    public partial class Bulletin
    {
        public Int32 BulletinId { get; set; }

        public DateTime DateCreated { get; set; }

        public String Topic { get; set; }

        public String Software { get; set; }

        public String Symptom { get; set; }

        public String Resolution { get; set; }

        public String Notes { get; set; }

        public String Noteimage { get; set; }

        public Boolean IsDeleted { get; set; }

        public DateTime DateModified { get; set; }
    }
}

File: alliance-bulletin/src/app/bulletin.ts
export class Bulletin {

    BulletinId: number = 0;
    DateCreated: Date = new Date(); ;
    Topic: String = '';
    Software: String = '';
    Symptom: String = '';
    Resolution: String = '';
    Notes: String = '';
    Noteimage: String = '';
    IsDeleted: Boolean = false;
    DateModified: Date = new Date();


    constructor(values: Object = {}) {
        Object.assign(this, values);
    }
}